python/scripts/consul/create_consul_watch.sh

```diff
@@ -0,0 +1,3 @@
+/bin/sh
+set -x
+consul watch -http-addr=http://${CONSUL_IP}:8500 -type keyprefix -prefix dev/ curl -H 'Content-Type:application/json' -X POST --data-binary @- http://${POD_IP}:8080/notify
```

Comment:
prefix should be a command line argument passed to this script?

Reply:
@dmetasoul01 Need status.hostIP & status.podIP after pod started. So set them in ENV from statefulset, get ENV during executing script in sidecar.

Reply:
> @dmetasoul01 Need status.hostIP & status.podIP after pod started. So set them in ENV from statefulset, get ENV during executing script in sidecar.

I mean the -prefix part, should be a parameter rather than fixed 'dev/'

Reply:
> > @dmetasoul01 Need status.hostIP & status.podIP after pod started. So set them in ENV from statefulset, get ENV during executing script in sidecar.
> 
> I mean the -prefix part, should be a parameter rather than fixed 'dev/'

oh, I will fix it by passing a parameter

Reply:
> #! is missing

thanks